charts/posthog/templates/pgbouncer-deployment.yaml

```diff
@@ -108,7 +108,7 @@ spec:
               #
               # See https://www.pgbouncer.org/usage.html
               #
-              command: ["/bin/sh", "-c", "killall -INT pgbouncer && sleep 60"]
+              command: ["/bin/sh", "-c", "pkill -INT pgbouncer && sleep 60"]
```

Comment:
@guidoiaquinti I wonder if the sleep is unnecessary? Presumably pgbouncer will exit at some point and the pod will be removed at that point? If it doesn't exit within the grace period it will be hard killed